cli/config/config.go

```diff
@@ -23,6 +25,23 @@ type Config struct {
 	Clusters []*Cluster `toml:"cluster"`
 }
 
+// copied from cli/main.go
+func homedir() string {
```

Comment:
Is there any reason not to use [`os/user.Current()`](http://golang.org/pkg/os/user/#Current) here?